packages/alpinejs/src/mutation.js

```diff
@@ -112,14 +112,16 @@ export function flushAndStopDeferringMutations() {
 }
 
 function onMutate(mutations) {
+    console.log(mutations);
```

Comment:
@calebporzio you forgot a console.log in the code here ;)

Reply:
@levu42 thanks! He removed it after in a WIP commit 😆